stories/assembly-aap-gcp-monitoring-logging.adoc

```diff
@@ -19,11 +19,10 @@ The Google Ops Agent is also responsible for the logging configuration.
 
```

Comment:
Link in line 12 points to old Tower docs: Updated link is
https://docs.ansible.com/automation-controller/latest/html/administration/metrics.html